crates/sui-types/src/messages.rs

```diff
@@ -50,8 +50,15 @@ use tap::Pipe;
 use thiserror::Error;
 use tracing::trace;
 
-// TODO: use RGP instead.
-pub const DUMMY_GAS_PRICE: u64 = 1;
+pub const TEST_ONLY_GAS_UNIT_FOR_TRANSFER: u64 = 2_000_000;
```

Comment:
OOC are these numbers you chose here based on anything specific?

Reply:
based on test failures :)